### PR TITLE
Align font awesome icons with the fa-fw class

### PR DESCRIPTION
**Pull request by viktorsmari**
Also minor code re-format. Put `a` in its own line

**Before**

**After**

### CODE REVIEW

**Review by tomage — APPROVED** [review]
Looks good to me!
There's some conflict in this file however.
I'll approve it, assuming the conflict is pretty basic, but I wonder if we should turn on "Require a pull request before merging" and even "Dismiss stale pull request approvals when new commits are pushed" in the branch protection settings in GitHub. Perhaps something to raise in an issue.

Here's the settings for the development branch, for reference: https://github.com/piratar/wasa2il/settings/branch_protection_rules/884889.

**Comment by tomage** [comment]
Did just look at the conflicts, and it does seem like the icons were mostly fixed in `development`. Might still be worth resolving the conflict, and merge in whatever is left :)

**Comment by viktorsmari** [comment]
Feel free to merge or close if the issue has already been fixed on `development`!

**Comment by tomage** [comment]
So.. I believe I have updated the patch with latest from development branch (I rebased your commit on latest development, and resolved the conflicts the best I could).

Really, all the patch is is:
1. Adding in `fa-fw` class wherever `fas` class is being used.
2. Formatted the HTML a bit, similar to the original patch (brought the <a>'s down into it's own level).

So I believe the patch is true to what it was before, but it might be good for someone to review it and give :+1:  - perhaps you've got time @viktorsmari ?

File: wasa2il/templates/_sidenav_polity.html
Comment (viktorsmari) [inline wasa2il/templates/_sidenav_polity.html]: Is this indentation correct?

Comment (tomage) [reply wasa2il/templates/_sidenav_polity.html]: Good catch! Fixed.

**Comment by viktorsmari** [comment]
LGTM :+1:

**Pull request merged**
